src/server/bnetserver/REST/LoginRESTService.cpp

```diff
@@ -247,52 +247,103 @@ int32 LoginRESTService::HandlePost(soap* soapClient)
 
     PreparedStatement* stmt = LoginDatabase.GetPreparedStatement(LOGIN_SEL_BNET_ACCOUNT_INFO);
     stmt->setString(0, login);
-    stmt->setString(1, CalculateShaPassHash(login, std::move(password)));
+
     if (PreparedQueryResult result = LoginDatabase.Query(stmt))
     {
+        std::string pass_hash = result->Fetch()[13].GetString();
```

Comment:
It references a table's field as `ip_address` line 204 so I did it to. Should I then modify it too or let `pass_hash` as this?